cmd/check.go

```diff
@@ -73,6 +142,10 @@ and/or users.`,
 				AllInstallModes:        checkflags.AllInstallModes,
 			}
 
+			if checkflags.ExtraCRDirectory != "" {
+				handleExtraCRDirectoryFlag(capAuditor)
```

Comment:
Should this instead be a part of capAuditor? 

```
capAuditor.ExtraCRDirectory(checkflags.ExtraCRDirectory)
```

Reply:
I really don't have a strong opinion here. I wasn't sure where to add this.

I followed you suggestion in ae2dfb333230741a232f7fd61edbfaf2927cdba0